UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/AzureMessageQueueHelper.cs

```diff
@@ -42,5 +42,18 @@ public async Task<HealthCheckResult> CheckMessageQueueHealth(string storageAccou
                 : HealthCheckResult.Unhealthy("Azure message queue is unhealthy", new Exception($"Azure message queue {queueName} does not exists"));
         
         }
+
+        public async Task AddMessage(string batchId, string storageAccountConnectionString, string message, string correlationId)
+        {
+            var queue = essFulfilmentStorageconfig.Value.ExchangeSetQueueName;
+            // Create the queue client.
+            QueueClient queueClient = new QueueClient(storageAccountConnectionString, queue);
+
+            // convert message to base64string          
+            var messageBase64String = System.Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(message));
```

Comment:
use using System.Text; line no. 31 as well.
```suggestion
            var messageBase64String = Convert.ToBase64String(Encoding.UTF8.GetBytes(message));
```

Reply:
Done the changes